controllers/authorino_controller.go

```diff
@@ -578,6 +636,39 @@ func (r *AuthorinoReconciler) createAuthorinoServices(authorino *api.Authorino)
 	return nil
 }
 
+func (r *AuthorinoReconciler) cleanupClusterScopedPermissions(ctx context.Context, crNamespacedName types.NamespacedName) {
+	crName := crNamespacedName.Name
+	sa := authorinoResources.GetAuthorinoServiceAccount(crNamespacedName.Namespace, authorinoServiceAccountName(crName))
```

Comment:
If we move `authorinoServiceAccountName` to the resource package, we just need to pass the crName and namespace to the `GetAuthorinoServiceAccount`

Reply:
Done.